query_rest.go

```diff
@@ -0,0 +1,61 @@
+package neutrino
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+)
+
+func (s *ChainService) queryRestPeers(
+	blockHash chainhash.Hash, query *cfiltersQuery) {
```

Comment:
Unused param `blockHash`, if we have everything we need in query I can remove it.